UnityProject/Assets/Scripts/Systems/Research/Techweb.cs

```diff
@@ -33,6 +48,56 @@ public List<TechWebNode> GetNodes()
 			return nodes;
 		}
 
+		public bool ResearchTechology(Technology technologyToResearch, bool updateUI = true)
+		{
+			if(researchPoints < technologyToResearch.ResearchCosts) return false;
+
+			ResearchedTech.Add(technologyToResearch);
+			researchedTechIDs.Add(technologyToResearch.ID);
+			researchPoints -= technologyToResearch.ResearchCosts;
+
+			UpdateTechnologyLists();
+			UpdateAvailableDesigns();
+			foreach(Technology tech in ResearchedTech)
+			{
+				Debug.Log(tech.DisplayName);
```

Comment:
Remove this log. If you need it inside a build use `Logger` instead.

Reply:
Yup accidentally left that in there from when I was debugging errors